crates/astria-conductor/src/sequencer.rs

```diff
@@ -52,34 +55,42 @@ use crate::{
         self,
         ClientProvider,
     },
+    executor,
 };
 
 pub(crate) struct Reader {
     /// The channel used to send messages to the executor task.
     executor_channel: mpsc::UnboundedSender<SequencerBlock>,
 
+    /// The state of the executor to receive information about the next
+    /// expected sequencer height.
+    executor_state: watch::Receiver<executor::State>,
+
+    /// The minimum height the executor must reach before this reader will forward
+    /// blocks to it.
+    not_before_height: Height,
```

Comment:
which height? `soft`?

Reply:
also this looks like it's only read from and never updated (unless i missed something)

Reply:
Yeah, I removed this in a subsequent PR. Sorry about that.